spk/rhash/src/installer.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+
+# Package
+PACKAGE="rhash"
+DNAME="rhash"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+
+
+preinst ()
+{
+    exit 0
+}
+
+postinst ()
+{
+    # Link
+    ln -s ${SYNOPKG_PKGDEST} ${INSTALL_DIR}
+    
+    # Put nano in the PATH
+    mkdir -p /usr/local/bin
+    ln -s ${INSTALL_DIR}/bini/rhash /usr/local/bin/rhash
```

Comment:
seems like you've got an typo in here shouldn't it be `bin` instead of `bini`

Reply:
Sorry, on and airport with shady ssh connection to my buildsystem so managed to sneak in some typos.